rllib/algorithms/dreamer/dreamer.py

```diff
@@ -201,52 +201,31 @@ def _postprocess_gif(gif: np.ndarray):
     return frames
 
 
-class EpisodicBuffer(object):
-    def __init__(self, max_length: int = 1000, length: int = 50):
-        """Stores episodes and samples chunks of size ``length`` from episodes.
+class EpisodeSequenceBuffer(ReplayBuffer):
+    def __init__(self, capacity: int = 1000, replay_sequence_length: int = 50):
+        """Stores episodes and samples sequences of size `replay_sequence_length`.
 
         Args:
-            max_length: Maximum episodes it can store
-            length: Episode chunking length in sample()
+            capacity: Maximum number of episodes this buffer can store
+            replay_sequence_length: Episode chunking length in sample()
         """
+        super().__init__(capacity=capacity, storage_unit=StorageUnit.EPISODES)
+        self.replay_sequence_length = replay_sequence_length
 
-        # Stores all episodes into a list: List[SampleBatchType]
```

Comment:
Mainly, the add method gets replaced by the standard add method for storage_unit=="episodes".
The rest stays the same.